example/src/app/app.module.ts

```diff
@@ -8,8 +8,8 @@ function initializeKeycloak(keycloak: KeycloakService) {
   return () =>
     keycloak.init({
       config: {
-        url: 'http://localhost:8080/auth',
         realm: 'master',
+        url: 'http://localhost:8080',
```

Comment:
Since v17.0.0, **Quarkus** version, the URL should not contain the *auth* value.